File: Rio/Rio.Web/Modules/ServerTypes/Membership/SignUpRequest.ts
import { ServiceRequest } from "@serenity-is/corelib";
import { CountryCode } from "../Web/Enums.CountryCode";

export interface SignUpRequest extends ServiceRequest {
    Email?: string;
    Password?: string;
    DisplayName?: string;
    FirstName?: string;
    LastName?: string;
    Countrycode?: CountryCode;
    Mobile?: string;
}

File: Rio/Rio.Web/Modules/Membership/Account/SignUp/SignUpRequest.cs
using Rio.Web.Enums;
using Serenity.Services;
using System;

namespace Rio.Membership
{
    public class SignUpRequest : ServiceRequest
    {
        public string Email { get; set; }
        public string Password { get; set; }
        public string DisplayName { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public CountryCode? Countrycode { get; set; }
        public string Mobile { get; set; }
    }
}